deeds-dapp-webapp/src/main/webapp/sitemap.xml

```diff
@@ -4,6 +4,11 @@
     <lastmod>2023-04-17</lastmod>
     <priority>1.00</priority>
   </url>
+  <url>
+    <loc>https://www.meeds.io/home</loc>
+    <lastmod>2023-04-17</lastmod>
+    <priority>1.00</priority>
+  </url>
```

Comment:
The same page is added twice in Sitemap, I think it's not recommended, can you verify this information please.

```suggestion
  <url>
    <loc>https://www.meeds.io/home</loc>
    <lastmod>2023-04-17</lastmod>
    <priority>1.00</priority>
  </url>
```

Reply:
fixed!